.gitattributes

```diff
@@ -0,0 +1,3 @@
+vendor/**                      linguist-vendored
```

Comment:
This avoid that GitHub load the diffs for files on this directory.

Reply:
``linguist-vendored`` does not exclude files from diffs. GH only excludes the files from stats.You have to use ``linguist-generated`` to hide the diffs.